pytest_subprocess/utils.py

```diff
@@ -127,7 +130,34 @@ def __init__(self, *, min: Optional[int] = None, max: Optional[int] = None) -> N
         self.max: Optional[int] = max
 
     def __str__(self) -> str:
-        return f"Any (min={self.min}, max={self.max})"
+        return f"{self.__class__.__name__} (min={self.min}, max={self.max})"
 
     def __repr__(self) -> str:
         return str(self)
+
+
+class Program:
+    """Specifies the name of the final program to be executed."""
+
+    def __init__(self, program: str) -> None:
+        self.program: str = program
+
+    def __repr__(self) -> str:
+        return f"{self.__class__.__name__}({self.program!r})"
```

Comment:
can you implement `__repr__()` and `__str__()` as it is in `Any` class?

Reply:
You mean with the space? Oh, I see, you implemented `repr` in terms of `str`. Any reason why? Python's default `__str__` calls `__repr__`, so if you want them the same, you can just implement `__repr__`. Plus this looks like a repr. Happy to change if you want it changed, just wondering if you have a special reason (like subclasses or something).

Reply:
Oh, it seems you're right. There was no special reason, I just thought you need to implement both. Now I know that I don't. So, no need to change that in your code.